src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categorySlice = createSlice({
+  name: 'category',
+  initialState: {
+    categories: [],
+    status: 'Under construction',
+  },
+  reducers: {
+    addCategory(state, action) {
+      state.categories.push(action.payload);
+    },
+    removeCategory(state, action) {
+      state.categories = state.categories.filter((category) => category !== action.payload);
```

Comment:
<img width="996" alt="image" src="https://github.com/wolterbwambale/bookstore-app/assets/27709832/be70d471-1f13-4b0c-aa8b-7a8f55677880">

The required reducers for the category slice should be `checkStatus` which should return the string `Under construction` always.